WalletWasabi.Gui/Global.cs

```diff
@@ -265,7 +265,10 @@ public static async Task InitializeNoWalletAsync()
 			{
 				if (Config.UseTor is true)
 				{
-					connectionParameters.TemplateBehaviors.Add(new SocksSettingsBehavior(Config.GetTorSocks5EndPoint(), onlyForOnionHosts: true));
+					// onlyForOnionHosts: false - Connect to clearnet IPs through Tor, too.
+					connectionParameters.TemplateBehaviors.Add(new SocksSettingsBehavior(Config.GetTorSocks5EndPoint(), onlyForOnionHosts: false));
+					// allowOnlyTorEndpoints: true - Connect only to onions and don't connect to clearnet IPs at all.
+					// This of course makes the first setting unneccessary, but it's better if that's around, in case someone wants to tinker here.
 					connectionParameters.EndpointConnector = new DefaultEndpointConnector(allowOnlyTorEndpoints: true);
```

Comment:
note that this mean you can't connect to clearnet peers

Reply:
I know. This facilitates end-to-end encryption. Otherwise the exit node could read and tamper the data. This way only the onions can lie to us about unconfirmed transactions, otherwise the exit nodes could be lying to us, too.  
Note, they cannot lie to us about blocks.

Reply:
I wonder if it's more performant or not.

On the one hand exit nodes are more loaded, on the other hand onions add 3-ish more hops to facilitate onion hiding.